src/odemis/gui/cont/streams.py

```diff
@@ -403,6 +488,32 @@ def _on_new_dye_name(self, dye_name):
         self.update_peak_label_fit(self._lbl_em_peak, self._btn_emission,
                                    self._dye_ewl, self.stream.emission.value)
 
+    def _on_metadata_btn(self, evt):
+        if isinstance(self.stream.raw[0], tuple):
+            # Show metadata for first element of raw in case of AR acquisitions or
```

Comment:
No need to show the metadata of each raw data in this case. They should all be identical (as they've been boken into tiles only for display reason). => do the same trick as above `md = self.stream.raw[0][0]`.

Reply:
Could you also put the check for `self.stream.raw` here? In such case, it should still show a window, but with a blank text. (Alternatively, you could just not display the button in such case, but I don't think it's a nice option as it could confuse the user, and it's better to be explicit that there is no data nor metadata).